model/src/main/kotlin/config/Excludes.kt

```diff
@@ -36,4 +40,14 @@ data class Excludes(
          */
         @JsonInclude(JsonInclude.Include.NON_EMPTY)
         val scopes: List<ScopeExclude> = emptyList()
-)
+) {
+    fun isExcluded(project: Project) = projects.any { it.path == project.definitionFilePath && it.exclude }
+
+    fun excludedScopes(project: Project): SortedSet<String> {
```

Comment:
Maybe rename this to `allExcludedScopes` to make the difference to the `scopes` property more clear?

Reply:
I think what you want is to make clear that this does not only takes the scopes from the Excludes class into account, but also from an optional project exclude, but I'm not sure if the word "all" would help here if having to provide a project doesn't. I wanted to add docs to the method, would that be sufficient for you?

Reply:
To be honest, I'm not convinced it would be sufficient, but due to a lack of better ideas for names (I was thinking of `excludedProjectScopes`, but that's again so long) I'm fine with sticking with better docs for now.